Hunting Queries/MultipleDataSources/NetworkConnectiontoOMIPorts.yaml

```diff
@@ -51,5 +54,12 @@ query: |
   | parse VM_s with * '/' VM 
   | project TimeGenerated, SourceIp = SrcIP_s, DestinationIp = DestIP_s, DestinationPort = DestPort_d, Protocol = L7Protocol_s, NSGRule_s, VM, Type
   | extend Timestamp = TimeGenerated, HostCustomEntity = VM, IPCustomEntity = SourceIp
+  ),
+  AzureDiagnostics
+  | where Category == "AzureFirewallNetworkRule" and OperationName == "AzureFirewallNatRuleLog"
+  | parse msg_s with Protocol ' request from ' SourceIp ':' SourcePort ' to ' DestinationIp ':' DestinationPort " was " Action " to " InternalIP ':' InternalPort
+  | where DestinationPort in (Port)
+  | project TimeGenerated, SourceIP, DestinationIP, DestinationPort, Protocol, Action, Resource
+  | extend Timestamp = TimeGenerated, HostCustomEntity = Resource, IPCustomEntity = SourceIp
```

Comment:
Seems like when we are mapping HostCustomEntity = Resource, we are mapping the host entity to the firewall that received the connection.  Wondering, if we really need the host entity mapping here or we need to remove it as it does not show the impacted host machine. Thanks.

Reply:
@aprakash13 

Thanks for reviewing and catching the issue with case, it has been updated now.  I have also removed the HostCustomEntity = Resource mapping.  The idea behind this was to show Firewall resource name in the query result but understand how this can be confusing for customers when they are expecting to see the actual host machine in this mapping.

Let me know if there is anything else we need to update else, please approve and merge.